jlm/llvm/Makefile.sub

```diff
@@ -65,70 +65,77 @@ libllvm_SOURCES = \
     jlm/llvm/opt/unroll.cpp \
 
 libllvm_HEADERS = \
```

Comment:
Holding this PR back until @haved merged the dot backend as it touches the files in the Makefile. I rather deal with the conflict here.